<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{border-image: url(:/pig/start.png);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>309</y>
      <width>469</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnNewGame">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>81</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
color:blue;
font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}
</string>
       </property>
       <property name="text">
        <string>登陆游戏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoad">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>81</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{

color:blue;
	font: 11pt &quot;方正舒体&quot;;
font-size:25px
}
QPushButton:hover {
    background-color: #008cba;
    color: white;
}</string>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>81</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover {
    background-color: #008cba;
    color: white;
}
QPushButton
{
color:blue;
	font: 11pt &quot;方正舒体&quot;;
font-size:25px
}</string>
       </property>
       <property name="text">
        <string>退出游戏</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>271</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:20px
}</string>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel
{
color:white;
font: 11pt &quot;方正舒体&quot;;
font-size:20px
}</string>
         </property>
         <property name="text">
          <string>密    码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPswd">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
